profiles/Adit0507.json

```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
+    "issueId": 1185,
+    "name": "Aditya Kumar Singh",
+    "username": "Adit0507"
+  }
+  
```

Comment:
Wrong indentation and missing an endline. This should fix it. 🙂
```suggestion
  "image": "https://user-images.githubusercontent.com/78037846/174435896-610e56c8-ecec-479b-b572-014a7873de6b.PNG",
  "issueId": 1185,
  "name": "Aditya Kumar Singh",
  "username": "Adit0507"
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix